23_map_destructuring/23_map_destructuring.cljs

```diff
@@ -0,0 +1,60 @@
+;; -=-=--=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-==--=
+;; An Introduction to ClojureScript for Light Table users
+;; -==-=--=-=-=-=-=-=-=-=-==-=--=-=-==--==-=-=-=-=--===--==-
+
+(ns lt-cljs-tutorial
+  (:require [clojure.string :as string]))
+
+
+;; Map destructuring
+;;-------------------
+
+;; Map destructuring is also useful. Here we destructure the value for the
+;; `:foo` key and bind ti to a local `f`, and the value for `:baz` key
+;; and bind it to a local `b`.
+
+(let [{f :foo b :baz} {:foo "bar" :baz "woz"}]
+  [f b])
+
+;; If we don't want to rename, we can just use `:keys`.
+
+(let [{:keys [first last]} {:first "Bob" :last "Smith"}]
+  [first last])
+
+;; We can also destructure a nested map.
+
+
+(let [{:keys [first last] {:keys [addr1 addr2]} :address} {:first "Bob" :last "Smith" :address {:addr1 "123" :addr2 "Main street"}}]
+  [first last addr1 addr2])
+
+
+;; Similar to :keys for keyword, :strs and :syms directies are available for matching string and symbol :keys.
```

Comment:
+;; Similar to :keys for keyword, :strs and :syms ~~directies~~ directives are available for matching string and symbol :keys.